File: Resources/Projects/Archive/2023/Competition/Programming/Contest/TOI-Zero_68/A2/019/TOI-Zero_68/A3/004/004.cpp
#include<bits/stdc++.h>
using namespace std;

#define ll long long

int main(){
	ios::sync_with_stdio(0);
	cin.tie(0);
	int n;
    cin >> n;
    vector<ll> xx,yy;
    pair<ll,ll> a[n];
    for(int i=0;i<n;i++){
        cin >> a[i].first >> a[i].second;
        ll xk=a[i].first+a[i].second;
        ll yk=a[i].second-a[i].first;
        a[i].first=xk;
        a[i].second=yk;
        xx.push_back(xk);
		yy.push_back(yk);
    }
    sort(xx.begin(),xx.end());
	sort(yy.begin(),yy.end());
    ll sx=xx[n/2];
	ll sy=yy[n/2];
	ll ans=0;
	for(int i=0;i<n;i++){
        ll x=a[i].first;
        ll y=a[i].second;
        ans+=abs(x-sx)+abs(y-sy);
    }
    cout << ans;
}

File: TOI-Zero_68/A3/004/004.py
n = int(input())
points = []
xx = []
yy = []

for i in range(n):
    x, y = map(int, input().split())
    xk = x + y
    yk = y - x
    points.append((xk, yk))
    xx.append(xk)
    yy.append(yk)

xx.sort()
yy.sort()
sx = xx[n//2]
sy = yy[n//2]

ans = 0
for x, y in points:
    ans += abs(x - sx) + abs(y - sy)

print(ans) 